docs/service-mesh/05-05-production-profile.md

```diff
@@ -22,18 +22,108 @@ The production profile introduces the following changes to the Istio Service Mes
 As the production profile is configured with increased performance it mind, the recommend system setup is different:
 
 | Requirement | Development setup | Production setup|
-|:--- | :--- | :--- | 
-| vCPU | 4 | 8 | 
-| RAM | 16 | 16/32 | 
-| Example machine type (GKE) | `n1-standard-4` | `n1-standard-8` or `c2-standard-8` | 
-| Example machine type (AKS) | `Standard_D4_v3` | `Standard_F8s_v2` or `Standard_D8_v3` | 
-| Size | 3 | 3-5 | 
+|:--- | :--- | :--- |
+| vCPU | 4 | 8 |
+| RAM | 16 | 16/32 |
+| Example machine type (GKE) | `n1-standard-4` | `n1-standard-8` or `c2-standard-8` |
+| Example machine type (AKS) | `Standard_D4_v3` | `Standard_F8s_v2` or `Standard_D8_v3` |
+| Size | 3 | 3-5 |
 
 ## Use the production profile
 
+>**CAUTION:** Istio is changing the installation options. Helm-based configuration is deprecated in favor of the new IstioControlPlane API. Please keep in mind that Helm overrides will be no longer supported in future Istio releases. Refer to [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb) page for more information.
+
 You can deploy a Kyma cluster with Istio configured to use the production profile, or configure Istio in a running cluster to use the production profile. Follow these steps:
 
 <div tabs>
+  <details>
+  <summary>
+  Istio Control Plane API
+  </summary>
+Istio installation in Kyma uses the [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb) API.
+This API is in the alpha version, but it's going to replace Helm-based approach in future Istio versions.
+Kyma provides a default IstioControlPlane configurations for local (Minikube) and cluster installations.
+You can add a custom control plane definition that overrides the default settings.
+The definition you provide may be a partial one (you don't have to specify all options), it will be merged with the defaults.
```

Comment:
```suggestion
The definition you provide may be a partial one (you don't have to specify all options). In that case it will be merged with the defaults.
```